heartbeat/monitors/active/http/config.go

```diff
@@ -49,6 +51,11 @@ type Config struct {
 	Check checkConfig `config:"check"`
 }
 
+type responseConfig struct {
+	IncludeBody         string `config:"include_body"`
+	IncludeBodyMaxBytes int    `config:"include_body_max_bytes"`
```

Comment:
Does also not seem to be documented. How does it overlap with the truncate? Probably see it in the code below.

Reply:
It is in the documentation! I documented the whole `response` block as one thing [here](https://github.com/elastic/beats/pull/13022/files#diff-2a01eeab73465de4b3125a1dbd932699R443). Glad to change it, but it's in keeping with the style of some other feature's documentation.

Reply:
Sorry, missed that. Seems my `command - F` foo failed me.